compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DocumentationAnalyzer.java

```diff
@@ -151,6 +152,17 @@ public void visit(BLangSimpleVariable varNode) {
         validateDeprecatedParametersDocumentation(varNode.markdownDocumentationAttachment, varNode.pos);
     }
 
+    @Override
+    public void visit(BLangTupleVariable tupleVariableNode) {
+        validateNoParameters(tupleVariableNode);
+        validateReturnParameter(tupleVariableNode, null, false);
+        validateReferences(tupleVariableNode);
+        validateDeprecationDocumentation(tupleVariableNode.markdownDocumentationAttachment, false,
+                tupleVariableNode.pos);
+        validateDeprecatedParametersDocumentation(tupleVariableNode.markdownDocumentationAttachment,
```

Comment:
Shall we add deprecation tests too? Let's check and add tests for each phase where possible, if not already done.